drivers/mfd/Kconfig.rp1

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Junho Lee <junho@tsnlab.com>
+# SPDX -License-Identifier: Apache-2.0
+
+config MFD_RP1
```

Comment:
can you help me to understand why is it an MFD driver

Reply:
RP1 is a peripheral controller for Raspberry Pi 5 and it allows RPi5 to use many peripherals such as GPIO, UART, etc. So I thought it fits best in the MFD category. On top of that, its driver is classified as an MFD driver in the Linux kernel.
https://github.com/raspberrypi/linux/blob/rpi-6.6.y/drivers/mfd/rp1.c

I guess you are asking this because there are only PCIe-related operations in the driver itself.

RP1 connects to the application processor via a PCIe bus, so PCIe initialization process is needed to use peripherals controlled by RP1.

Current implementation of this GPIO driver does not need any other features provided by RP1, so only PCIe initialization process is included in this driver.

More features will be added when other peripherals that need them are implemented.

Reply:
Since it is also listed as mfd in the Linux device tree, I don't think there is any problem with this classification.